db/controllers/endpoints-controller.js

```diff
@@ -0,0 +1,12 @@
+const fs = require("fs/promises");
+
+const getEndpoints = (req, res, next) => {
+  fs.readFile(
+    "endpoints.json"
+  ).then((response) => {
+    const JSONres = JSON.parse(response);
+    res.status(200).send(JSONres)
+  });
+};
```

Comment:
Nicely done, note this can also be handled by requiring in the file (as it is a JSON, it's accessible immediately)